src/features/offer/components/__tests__/OfferHeader.test.tsx

```diff
@@ -130,7 +139,7 @@ describe('<OfferHeader />', () => {
     expect(getByTestId('icon-favorite-filled')).toBeTruthy()
   })
 
-  it('should add favorite when adding an offer in favorite - logged in users', async () => {
+  it.only('should add favorite when adding an offer in favorite - logged in users', async () => {
```

Comment:
si les test est vide, pourquoi mettre only ?

est-ce que tu peux le remplacer par it.skip() de manière à en garder une trace plus facilement ?

Reply:
Oubli, OK

Reply:
Oubli, OK